ujenkins/endpoints/builds.py

```diff
@@ -21,10 +21,41 @@ class Builds:
     def __init__(self, jenkins) -> None:
         self.jenkins = jenkins
 
-    def get(self, name: str) -> List[dict]:
+    def get(self,
+            name: str,
```

Comment:
also I suggest to add * delimiter, to make it impossible to pass as args but kwargs